src/components/home/searchFiltering/filteringContents/FilteringContents.tsx

```diff
@@ -49,6 +50,8 @@ export default function FilteringContents() {
               width='90%'
               selectSkills={selectSkills}
               setSelectSkills={setSelectSkills}
+              reset={true}
+              onHandleSkillTagReset={handleSkillTagReset}
```

Comment:
```suggestion
              reset
              onHandleSkillTagReset={handleSkillTagReset}
```
boolean값은 값을 넣지 않아도 나타낼 수 있기도 하죠.
props의 네이밍을 조금 더 고려해보면 좋을 것 같아요.
reset이 가능한지 여부에 대한 props니까요!
handler도 onReset정도로 갈무리할 수 있을지도요 👀